pages/core/faq.mdx

```diff
@@ -30,6 +30,6 @@ The bridging process can take a long time. You generally have to wait some hours
 
 1, 2, 3, and 4 characters names have higher pricing to reflect the small number of these names available and to prevent squatting, 5+ letters domains are way cheaper.
 
-![domain-prices](public/illustrations/prices.webp)
+![domain-prices](public/illustrations/new_prices.webp)
```

Comment:
We made a mistake please change the 1 letter price to `0.29 ETH`, sorry for that please remake another image :/